src/state/lists/updater.ts

```diff
@@ -20,15 +21,15 @@ export default function Updater(): null {
   const fetchList = useFetchListCallback()
   const fetchAllListsCallback = useCallback(() => {
     if (!isWindowVisible) return
-    Object.keys(lists).forEach((url) => {
+    DEFAULT_LIST_OF_LISTS.forEach((url) => {
       // Skip validation on unsupported lists
       const isUnsupportedList = UNSUPPORTED_LIST_URLS.includes(url)
-      fetchList(url, false, isUnsupportedList).catch((error) => console.debug('interval list fetching error', error))
+      fetchList(url, isUnsupportedList).catch((error) => console.debug('interval list fetching error', error))
     })
-  }, [fetchList, isWindowVisible, lists])
+  }, [fetchList, isWindowVisible])
 
   // fetch all lists every 10 minutes, but only after we initialize provider
-  useInterval(fetchAllListsCallback, provider ? 1000 * 60 * 10 : null)
+  useInterval(fetchAllListsCallback, provider ? ms`10m` : null)
```

Comment:
@zzmp I noticed useInterval will trigger fetchAllListsCallback again if fetchAllListsCallback updates. Is this behavior
 expected? It's new since [WEB-2004](https://github.com/Uniswap/interface/tree/WEB-2004) and was causing a inf looping condition until I refactored to use `DEFAULT_LIST_OF_LISTS` instead of `lists`

It's not an issue here now, was just curious if that's expected or if its a bug in useInterval

Reply:
This _could_ be considered a bug, but not necessarily.

interface's useInterval will retrigger if the callback changes - the callback is expected to be memoized - see https://github.com/Uniswap/interface/blob/main/src/lib/hooks/useInterval.ts
widget's useInterval will never retrigger - see https://github.com/Uniswap/widgets/blob/main/src/hooks/useInterval.ts

I think both can be considered valid, and have different pros/cons. eg widget's will not auto-update if something changes, you'd need to wait the full 10 minutes.

Reply:
Makes sense, maybe we'll consolidate them eventually